src/wallet/rpcwallet.cpp

```diff
@@ -4764,6 +4765,8 @@ size_t EstimateTxSize(
         mtx.nVersionGroupId = ZIP225_VERSION_GROUP_ID;
         mtx.nVersion = ZIP225_TX_VERSION;
     }
+    // Fine to call this because we are only testing that `mtx` is a valid size.
+    mtx.saplingBundle = sapling::test_only_invalid_bundle(0, saplingRecipientCount, 0);
```

Comment:
> We have to link all the test_only_ stuff into zcashd, etc. for now because of the limitations with the bridge, right? (I.e., only being able to have one bridge)

Correct. There may be some pieces we can break out into separate bridges, but any test code that needs to produce Rust types from the main bridge, itself needs to be in the main bridge. It shouldn't get linked into `zcashd` unless it is actually used however, because we statically link the Rust code in and perform link-time optimization.

> But maybe we can encapsulate it a bit better?

I'll play with this a bit and see if I can improve the namespacing. Not going to block on doing so though because of the above limitations on what can be done here.

Reply:
I didn't come up with anything nicer in the bridge file. Leaving this to future refactors.